lib/graphql/schema/member/type_system_helpers.rb

```diff
@@ -17,15 +17,24 @@ def to_non_null_type
         end
 
         # @return [Schema::List] Make a list-type representation of this type
-        def to_list_type
-          @to_list_type ||= GraphQL::Schema::List.new(self)
+        def to_list_type(skip_nodes_on_raise: false)
```

Comment:
I used a default value here, because there's a fair number of callsites the way `Addition#update_type_owner` calls this reflectively can't pass an extra parameter (without changing it a fair bit)


    https://github.com/amomchilov/graphql-ruby/blob/f88ec1ffa70208795c7a28f7f66a1f39e6ed5a40/lib/graphql/schema/addition.rb#LL135C1-L135C1